docs/usage/validator-management.md

```diff
@@ -36,58 +36,42 @@ Validators are represented by a BLS keypair. Use your generated mnemonic from on
 
 To import a validator keystore that was created via one of the methods described above, you must locate the validator keystore JSONs exported by those tools (ex. `keystore-m_12381_3600_0_0_0-1654128694.json`).
 
-Inside the keystore JSON file, you should have an [EIP-2335 conformant keystore file](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2335.md#json-schema) such as the example below:
+Inside the keystore JSON file, you should have an [EIP-2335 conformant keystore file](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2335.md#json-schema).
 
-```
-{
-  "crypto": {
-    "kdf": {
-      "function": "scrypt",
-      "params": {
-        "dklen": 32,
-        "n": 262144,
-        "r": 8,
-        "p": 1,
-        "salt": "30bb9ef21d9f1f946c3c7ab70e27f453180a49d473a2a3e79ca2bc715ac4e898"
-      },
-      "message": ""
-    },
-    "checksum": {
-      "function": "sha256",
-      "params": {},
-      "message": "ba3cf1c8ba5be4f90c36bcf44ee37a779eac8c54b72121e4755b6722e95164a7"
-    },
-    "cipher": {
-      "function": "aes-128-ctr",
-      "params": {
-        "iv": "90f76d9d4d1b089e89802eac2f80b6b7"
-      },
-      "message": "8de2b0f55da54719822db6c083f0436ff94cd638be96c57b91339b438e9355f6"
-    }
-  },
-  "description": "",
-  "pubkey": "b22690ca679edd5fb9c2545f358da1427b8310e8ccf9e7e4f01ddce9b1d711a0362d35225673cce8f33911a22ae1519e",
-  "path": "m/12381/3600/0/0/0",
-  "uuid": "de83e8dc-8f95-4ea0-b9ba-cfa608ff3483",
-  "version": 4
-}
-```
+You will also need the passphrase used the encrypt the keystore. This can be specified interactively, or provided in a plaintext file.
 
-These keystore files can be imported into your Lodestar's keystores folder with the `--importKeystores` command.
+#### Import Keys To Secret Store
 
+You can load the keys into the secret store using the command `validator import`:
 ```bash
-./lodestar validator \
-  --importKeystores keystore-m_12381_3600_0_0_0-1654128694.json
+# Interactive passphrase input
+./lodestar validator import --keystore ./validator_keys
+
+# Plaintext passphrase file input
+./lodestar validator import --keystore ./validator_keys --passphraseFile ./password.txt
 ```
+These will be automatically loaded at runtime.
```

Comment:
Instead of runtime, I would say when running the validator command specifically

Reply:
We do need to be careful not to confuse people here as we are running the validator command to import these keys, but I think this can be phrased as "running the validator". Will update.